packages/contents/src/drivefs.ts

```diff
@@ -343,6 +357,7 @@ export class ContentsAPI {
 
   readdir(path: string): string[] {
     const dirlist = this.request({
+      sender: 'drivefs.ts',
```

Comment:
Could this be a `const SENDER = 'drivefs.ts'` at the top of the file?

Reply:
Fixed that now